### PR TITLE
Artifact and Model CRUD Testing

### PR DESCRIPTION
**Pull request by vitorbezerra**
## Features:

* Add Github Actions;
* Add Artifacts CRUD Pytest;
* Add Models CRUD Pytest;

### CODE REVIEW

File: test/local/CRUD/test_artifacts.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+"""
+Unit tests for the entity Artifact
+"""
+
+from digitalhub_core.entities.artifact.entity._base import Artifact
```
Comment (EvangMM) [inline test/local/CRUD/test_artifacts.py:5]: Just change digitalhub_core in digitalhub

File: test/local/CRUD/test_models.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+"""
+Unit tests for the entity Models
+"""
+
+from digitalhub_ml.entities.model.entity._base import Model
```
Comment (EvangMM) [inline test/local/CRUD/test_models.py:5]: Idem here, from digitalhub

File: test/local/CRUD/test_artifacts.py (hunk 1 above)
Comment (vitorbezerra) [reply test/local/CRUD/test_artifacts.py]: Changed

File: test/local/CRUD/test_models.py (hunk 2 above)
Comment (vitorbezerra) [reply test/local/CRUD/test_models.py]: Changed

**Pull request merged**